src/core/commandVersionUpgrades/upgradeV1ToV2/upgradeV1ToV2.ts

```diff
@@ -41,7 +41,7 @@ function upgradeModifier(modifier: ModifierV0V1): ModifierV2[] {
         {
           type: includeSiblings ? "everyScope" : "containingScope",
           scopeType: {
-            type: scopeType as SimpleScopeTypeType,
+            type: scopeType as SimpleScopeTypeTypeV2,
```

Comment:
Can we get rid of this type cast somehow?

Reply:
Possibly, but doesn't seem like a huge deal.  We should probably figure out a better migration setup as part of #983, and we should consider not pinning scope type list during migrations at that point.  But for now this doesn't strike me as a big loophole, but happy to give it a bit more thought if you think it's a risk